cmd/exo/README.md

```diff
@@ -25,7 +25,12 @@ $ go install
 ## Configuration
 
 A configuration file holding your credentials is required.
-You can generate one via a guided prompt.
+You can generate one via a guided interactive prompt with:
+
+```
+$ exo config
+```
+
```

Comment:
as it duplicates what's below under _automatic configuration_, why not removing the _You can generate..._ line?

Reply:
Fair enough :)

Reply:
Actually, how about moving the "Automatic configuration" header to be "configuration"?

/me does